src/commands/kv/bulk/mod.rs

```diff
@@ -1,37 +1,50 @@
-extern crate base64;
+mod delete;
+mod write;
 
-use cloudflare::framework::apiclient::ApiClient;
-
-use std::fs;
-use std::fs::metadata;
-use std::path::Path;
+pub use delete::delete;
+pub use write::write;
 
+use cloudflare::endpoints::workerskv::delete_bulk::DeleteBulk;
 use cloudflare::endpoints::workerskv::write_bulk::KeyValuePair;
 use cloudflare::endpoints::workerskv::write_bulk::WriteBulk;
+use cloudflare::framework::apiclient::ApiClient;
 
 use crate::commands::kv;
+use crate::message;
 use crate::settings::global_user::GlobalUser;
 use crate::settings::project::Project;
-use crate::terminal::message;
 
 const MAX_PAIRS: usize = 10000;
 
-pub fn write_json(
+fn delete_bulk(
     project: &Project,
     user: GlobalUser,
     namespace_id: &str,
-    filename: &Path,
+    keys: Vec<String>,
 ) -> Result<(), failure::Error> {
-    let pairs: Result<Vec<KeyValuePair>, failure::Error> = match metadata(filename) {
-        Ok(ref file_type) if file_type.is_file() => {
-            let data = fs::read_to_string(filename)?;
-            Ok(serde_json::from_str(&data)?)
-        }
-        Ok(_) => failure::bail!("{} should be a JSON file, but is not", filename.display()),
-        Err(e) => failure::bail!(e),
-    };
+    let client = kv::api_client(user)?;
+
+    // Check number of pairs is under limit
+    if keys.len() > MAX_PAIRS {
+        failure::bail!(
+            "Number of keys to delete ({}) exceeds max of {}",
+            keys.len(),
+            MAX_PAIRS
+        );
+    }
 
-    write_bulk(project, user, namespace_id, pairs?)
+    let response = client.request(&DeleteBulk {
+        account_identifier: &project.account_id,
+        namespace_identifier: namespace_id,
+        bulk_keys: keys,
+    });
+
+    match response {
+        Ok(_success) => message::success("Success"),
+        Err(e) => kv::print_error(e),
+    }
+
+    Ok(())
 }
 
 fn write_bulk(
```

Comment:
I think it would be good to move this to write.rs and make it public, given that buckets will use it in the future :)